nipype/interfaces/spm/model.py

```diff
@@ -474,15 +474,16 @@ def _make_matlab_command(self, _):
             script += "condnames=names;\n"
         else:
             if self.inputs.use_derivs:
-                script += "pat = 'Sn\([0-9]*\) (.*)';\n"
+                script += r"pat = 'Sn\([0-9]*\) (.*)';" "\n"
```

Comment:
Hopefully we test these. I'm less comfortable with these, since they need to work when written to MATLAB scripts.

Reply:
as long as we are still running the spm workflows.

Reply:
Codecov doesn't seem to hit them.

Reply:
I've tested:

```Python
>>> a = ""
>>> a += r"\s+" "\nnextline"                                                                   
>>> print(a)                                                                                   
\s+
nextline
```

So rawstring/string concatenation works fine, and the `+=` is lower precedence, so I'm reasonably confident in this change.

Merging.